skbio/io/format/stockholm.py

```diff
@@ -577,3 +576,79 @@ def _check_for_malformed_line(line, expected_len):
         raise StockholmFormatError('Line contains %d item(s). It must '
                                    'contain exactly %d item(s).'
                                    % (len(line), expected_len))
+
+
+@stockholm.writer(TabularMSA)
+def _tabular_msa_to_stockholm(obj, fh):
+    index = list(obj.index)
+    # Writes header
+    fh.write("# STOCKHOLM 1.0\n")
+
+    # Writes GF data to file
+    if obj.has_metadata():
+        for i in range(len(obj.metadata)):
+            gf_feature = list(obj.metadata.keys())[i].strip("'")
+            if gf_feature == 'NH' and (type(obj.metadata[gf_feature]) is
```

Comment:
Can you add a unit test that would have caught this? i.e. making sure you can write a single tree with identifier that is stored in a `dict` instead of `OrderedDict`